src/QirRuntime/lib/Tracer/tracer.hpp

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <limits>
+#include <memory>
+#include <unordered_map>
+#include <vector>
+
+#include "CoreTypes.hpp"
+#include "QuantumApi_I.hpp"

```

Comment:
Reorder includes?

Reply:
Personally, I like grouping std includes and the project includes separately, and inside each group they are sorted alphabetically by clang-format, which improves readability and is usually sufficient to catch the header dependencies. My preference to list std includes first comes from the desire to keep "meaningful" code together. In my previous projects, std includes were seldom the source of missing header dependencies and they are easy to fix, if found. All in all, I'd like to keep the current format/order of includes as is, and consistent across QirRuntime project.

Reply:
If you feel strongly about it, please add to https://github.com/microsoft/qsharp-runtime/issues/482 (we'll need to update our clang.format and make sure we use consistent grouping/ordering of includes across the project).